index.js

```diff
@@ -102,3 +100,50 @@ module.exports.isMACValid = function(mac) {
 
     return !(mac.length != 2 * MAC_BYTES || mac.match(/[^a-fA-F0-9]/));
 }
+
+module.exports.relay = function(mac, params) {
+
+        getIP(params).then(function(ip) {
```

Comment:
we can extend `wake` method. for example:

```js
module.exports = function wake(...) {
    getIP(params).then(function(ip) {
         if (params.relay) {
             wakeThroughRelay();
         } else {
             wake();
         }
    });
};
```

Reply:
Ok